### ISSUE TITLE
New Plugin: wp-slack-jetpack

### ISSUE THREAD

**Issue by bhubbard** [gedex/wp-slack#23]
Hey I am posting this here, sorry if its not the best way to reach out for such a request. I would love to see a add-on plugin for this that integrates with Jetpack. Here are some possible uses:
- Post message in slack when site goes up or down based on jetpack monitor feature
- Post message in slack when post is liked or shared using Jetpack Sharing & Like Features
- Post message in slack when post is published on Facebook/Twitter/etc when Publicized is used
- Post message in slack (time based via WP Cron) to display website stats such as visitor counts

**Comment by harry61286** [comment]
+1 Post message in slack when site goes up or down based on jetpack monitor feature

**Comment by gedex** [comment]
I don't think this is possible with this plugin to send notification when site goes down -- the plugin relies on the site to be up. However, you can relay your mail from no-reply@jetpack.me (see https://jetpack.me/support/monitor/) to your incoming integration -- not sure if there's a service for that.

For other ideas mentioned above, it might be possible. Need to dive into Jetpack hooks though.
